package.json

```diff
@@ -3157,7 +3157,7 @@
                 "command": "aws.codeWhisperer",
                 "key": "alt+c",
                 "mac": "alt+c",
-                "when": "editorTextFocus"
+                "when": "editorTextFocus && CODEWHISPERER_ENABLED"
```

Comment:
Unrelevant to the bug. But it will fix: when codewhisperer in disabled, users are able to press option + c and invoke CW service getting recommendation, the recommendations will not be rendered tho.

Reply:
when codewhisperer is disabled, it shouldn't even get suggestions correct?

Reply:
yes correct